extensions/amp-form/amp-form.md

```diff
@@ -129,6 +129,11 @@ The value for `action-xhr` can be the same or a different endpoint than `action`
 
 To learn about redirecting the user after successfully submitting the form, see the [Redirecting after a submission](#redirecting-after-a-submission) section below.
 
+##### xssi-prefix (optional)
+
+Specifies that a prefix should be stripped prior to parsing the fetched json from the `action-xhr` endpoint. If the prefix is not present in the response, then this attribute will have no effect.
+This can be useful for APIs that include [security prefixes](http://patorjk.com/blog/2013/02/05/crafty-tricks-for-avoiding-xssi/) like `)]}` to help prevent cross site scripting attacks.
+
 ##### data-initialize-from-url
```

Comment:
Minor: I think this should be "(optional)" too. Then I'd move `xssi-prefix` below it.

Reply:
Done